spinalcordtoolbox/image.py

```diff
@@ -220,12 +221,44 @@ def __getitem__(self, idx):
         return [x[idx] for x in self.slicers]
 
 
+def check_affines_match(im):
+    hdr = im.hdr
+    hdr2 = hdr.copy()
+
+    try:
+        hdr2.set_qform(hdr.get_sform())
+    except np.linalg.LinAlgError:
+        if im.absolutepath is None:
+            logger.error("Internal code has produced an image with inconsistent qform and sform "
+                         "please report this on github at https://github.com/neuropoly/spinalcordtoolbox/issues "
+                         "or on the SCT forums https://forum.spinalcordmri.org/.")
+
+        logger.warning("The sform for {} is uninitialized and may cause unexpected behaviour."
+                       ''.format(im.absolutepath))
+        return(True)
```

Comment:
Redundant section of code -- are both logger errors necessary?

Reply:
Ah I see now. It's not redundant -- it was just copied and pasted but not edited.